src/SDKs/SqlManagement/Sql.Tests/Utilities/SqlManagementTestUtilities.cs

```diff
@@ -26,24 +26,74 @@ namespace Sql.Tests
 {
     public class SqlManagementTestUtilities
     {
-        public const string DefaultLocationId = "japaneast";
+        private static readonly TestEnvironment _environment = new TestEnvironment(Environment.GetEnvironmentVariable("TEST_CSM_ORGID_AUTHENTICATION"));
 
-        public const string DefaultLocation =  "Japan East";
+        // We now load default locations from environment variable.
+        // Tests were recorded with this appended to DefaultLocationId=japaneast;DefaultLocation=Japan East;DefaultSecondaryLocationId=centralus;DefaultSecondaryLocation=Central US;DefaultStagePrimaryLocation=North Europe;DefaultStageSecondaryLocation=SouthEast Asia;DefaultEuapPrimaryLocation=East US 2 EUAP;DefaultEuapPrimaryLocationId=eastus2euap
+        public static string DefaultLocation
+        {
+            get
+            {
+                return _environment.ConnectionString.KeyValuePairs["DefaultLocation"];
```

Comment:
what happens when you run in playback mode and your connection string is empty? will this throw keynotfoundexception? I suggest using TestEnvironmentUtilities.GetOrAddVariable